docs/user/dashboard/lens.asciidoc

```diff
@@ -139,6 +139,42 @@ image::images/lens_drag_drop_3.gif[Using drag and drop to reorder]
 
 . Press Space bar to confirm, or to cancel, press Esc.
 
+[float]
+[[lens-formulas]]
+==== Formulas
```

Comment:
Can we make this title actionable? Something like, `Perform math on aggregated data` or `Use formulas to perform math`.